Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -334,6 +347,97 @@ private Branch(Context context) {
 	}
 
 	/**
+	 *
+	 * <p>Creates and initialises a new  singleton instance of object of type {@link Branch} if
+	 * not existing.Application developers should not call this method directly.Instead extend
+	 * your Application class with {@link BranchLinkedApp} or use {@link BranchLinkedApp} as the
+	 * Application class in your manifest.
+	 * You need to set Branch key in your manifest</p>
+	 *
+	 * @see          <a href="https://github.com/BranchMetrics/Branch-Android-SDK/blob/05e234855f983ae022633eb01989adb05775532e/README.md#add-your-app-key-to-your-project">
+	 *                 Adding your app key to your project</a>
+	 *
+	 * @param application  Your {@link Application} instance.
+	 *
+	 */
+
+	public static void createInstance(Application application) {
+		isBranchAppInstantiated = application instanceof  BranchLinkApp;
+      /* Create and initialise a new instance of Branch. */
+		Branch createdInstance = getBranchInstance(application,true);
```

Comment:
So, in android Application and Context are the same thing? Current getBranchInstance accepts Context as its first param, and I don't see it change here.